babel.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {
+      include: [
+        // required for create-react-app
+        '@babel/plugin-proposal-nullish-coalescing-operator'
```

Comment:
Would be nice to specify the upper es version instead, https://github.com/babel/babel/issues/6613